modules/react/tooltip/lib/TooltipContainer.tsx

```diff
@@ -72,9 +85,9 @@ export const TooltipContainer = styled('div')<TooltipContainerProps>(
 
     // offset tooltips by 2 pixels when a keyboard focus ring is detected
     '[data-whatinput=keyboard] &': {
```

Comment:
do we still need whatinput here?

Reply:
Not totally sure honestly. If we're looking for keyboard specifically, then yes I would imagine so.